docs/content/guides/author-apps/azure/howto-azure-container-instances/index.md

```diff
@@ -0,0 +1,115 @@
+- Define and deploy the demo application to the Environment using Radius to provision the necessary resources to run the application containers in ACI.
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- Radius installed and initiated on a [supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+- Azure provider configured and registered with your Radius control plane, either through [Service Principal](https://docs.radapp.io/guides/operations/providers/azure-provider/howto-azure-provider-sp/) or [Workload Identity](https://docs.radapp.io/guides/operations/providers/azure-provider/howto-azure-provider-wi/)
+- A [user-assigned managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp) that has been [assigned](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal-managed-identity) to the `Contributor` and `Azure Container Instances Contributor` roles on the subscription or resource group where the ACI will be deployed
+
+## Step 1: Define the Environment with ACI compute
+
+Create a new file named `app.bicep` and add the following Environment definition. This Environment uses the `aci` compute provider and a user-assigned managed identity to provision the necessary resources for ACI and also registers a default Recipe for provisioning an Azure Redis Cache.
+
+{{< rad file="snippets/app.bicep" embed=true marker="//ENVIRONMENT">}}
+
+> Note: be sure to replace the `resourceGroup` and `scope` values with your resource group ID and the `managedIdentity` value with your managed identity ID.
+
+## Step 2: Deploy the Environment
+
+1. Run the following command to deploy the Environment:
+   ```bash
+   rad deploy ./app.bicep --resource-group aciGroup
+   ```
+
+   You should see the following terminal output:
+
+   ```
+   Deployment In Progress...
+
+   Deployment Complete
+
+   Resources:
+   aci-demo Applications.Core/environments
+   ```
+
+## Step 3: Define the Application and its resources
+
+Add the application definition, along with Redis cache and gateway resources to the `app.bicep` file.
+
+{{< rad file="snippets/app.bicep" embed=true marker="//APPLICATION" >}}
+
+Add the application Container resource to the `app.bicep` file.
+
+{{< rad file="snippets/app.bicep" embed=true marker="//CONTAINER" >}}
+
+> Notice that for ACI containers, you define a Gateway resource that provides L7 traffic for the container. Radius will provision the Gateway in Azure on your behalf and configure the container to use the Gateway as its ingress. The Gateway will be provisioned with a public IP address and a DNS name that you can use to access the application.
```

Comment:
We should add that the alternative is an internal load balancer is used for just L4 traffic. An ILB is the default setup right now but if a gateway is provided, we'll use the gateway as the network profile for ACI instead

Reply:
I think I'll leave out the alternative for the moment to avoid confusing the user too much in this how-to guide.